.gitignore

```diff
@@ -94,17 +94,7 @@ ENV/
 .ropeproject
 
 # User-specific stuff:
-.idea/workspace.xml
-.idea/tasks.xml
-
-# Sensitive or high-churn files:
-.idea/dataSources
-.idea/dataSources.ids
-.idea/dataSources.xml
-.idea/dataSources.local.xml
-.idea/sqlDataSources.xml
-.idea/dynamic.xml
-.idea/uiDesigner.xml
+.idea/*
```

Comment:
PyCharm allows for sharing configuration, as explained here: https://www.jetbrains.com/help/pycharm/synchronizing-and-sharing-settings.html#d321724e151

This should make collaborators' lifes easier.

Did you have any issue with this?